README.md

```diff
@@ -26,9 +130,30 @@ $ ./scripts/run.sh populate_with_test_data.py
 
 * Run the script to import the Kanban dashboard to the new collection. Make sure to use the proper username and collection-id:
 ```
-./scripts/metabase-import-export.py \
+metabase-import-export \
     --username=<your-user-name> \
     import \
     --collection-id=<your-collection-id> \
-    --import-file=kanban-dashboards.json
+    --import-file=kanbandash/kanban-dashboards.json
+```
+
+or using our docker container:
+
+```
+docker run kanban-dash metabase-import-export \
```

Comment:
Fixing the image name:
```suggestion
docker run cravefood/kanban-dash metabase-import-export \
```

Reply:
Reminder: We have to push the new docker containers ;)

![image](https://user-images.githubusercontent.com/9268203/103369919-24f71380-4aaa-11eb-8d70-fe8c4abba53e.png)